src/app/my-account/my-account.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="center">
+  <h1>My Account</h1>
+  <form style="width: 75%; display: inline-block">
+    <div [ngClass]="{
+      blue: passColors.oldPass === this.colors.blue,
+      green: passColors.oldPass === this.colors.green,
+      red: passColors.oldPass === this.colors.red,
+      group: true
+    }">
+      <label for="oldPassword" class="group-start">Old Password/Passphrase:</label>
+      <input [(ngModel)]="passFields.oldPassField" (input)="oldPassChange()" type="password" id="oldPassword"
+        class="group-end" name="oldPassword" placeholder="Old Password...">
+    </div>
+
+    <div>
+      <h2>You may use a password or passphrase.</h2>
+      <h3>A password must be at least 8 characters long and include <strong><em>at least 3</em></strong> character types.</h3>
+      <h3>A passphrase must be at least 15 characters long and include <strong><em>at least 1</em></strong> character type.</h3>
```

Comment:
I'd explain that passphrases are long but easy-to-remember phrases or sentences (but not famous quotes).

Reply:
I suggest using a passphrase and give a brief explanation with examples.